rules/core.js

```diff
@@ -0,0 +1,99 @@
+    "comment-whitespace-inside": "always",
+    /* Urls must always be quoted. */
+    /* https://stylelint.io/user-guide/rules/function-url-quotes */
+    "function-url-quotes": "always",
+    /* Strings must always be wrapped with single quotes. */
+    /* https://stylelint.io/user-guide/rules/string-quotes */
+    "string-quotes": "single",
+    /* Disallow duplicate selectors within a stylesheet. */
+    /* https://stylelint.io/user-guide/rules/no-duplicate-selectors */
+    "no-duplicate-selectors": true,
+    /* Limit the number of decimal places allowed in numbers. */
+    /* https://stylelint.io/user-guide/rules/number-max-precision */
+    "number-max-precision": 2,
+    /* Disallow trailing zeros in numbers. */
+    /* https://stylelint.io/user-guide/rules/number-no-trailing-zeros */
+    "number-no-trailing-zeros": true,
+    /* Require or disallow a leading zero for fractional numbers less than 1. */
+    /* https://stylelint.io/user-guide/rules/number-leading-zero */
+    "number-leading-zero": "never",
+    /* Disallow redundant values in shorthand properties. (e.g. margin / padding) */
+    /* https://stylelint.io/user-guide/rules/shorthand-property-no-redundant-values */
+    "shorthand-property-no-redundant-values": true,
+    /* Disallow qualifying a selector by type. Allow attribute selectors qualified by type. */
+    /* https://stylelint.io/user-guide/rules/selector-no-qualifying-type */
+    "selector-no-qualifying-type": [true, { ignore: "attribute" }],
+    /* Require a newline or disallow whitespace after the commas of selector lists. */
+    /* https://stylelint.io/user-guide/rules/selector-list-comma-newline-after */
+    "selector-list-comma-newline-after": null,
+    /* Limit the number of ID selectors in a selector */
+    /* https://stylelint.io/user-guide/rules/selector-max-id */
+    "selector-max-id": 0,
+    /* Limit the number of universal selectors in a selector. */
+    /* https://stylelint.io/user-guide/rules/selector-max-universal */
+    "selector-max-universal": 0,
+    /* Disallow vendor prefixes for selectors. */
+    /* https://stylelint.io/user-guide/rules/selector-no-vendor-prefix */
+    "selector-no-vendor-prefix": true,
+    /* Disallow vendor prefixes for properties. */
+    /* https://stylelint.io/user-guide/rules/property-no-vendor-prefix */
+    "property-no-vendor-prefix": true,
+    /* Specify lowercase or uppercase for media feature names. */
+    /* https://stylelint.io/user-guide/rules/media-feature-name-case */
+    "media-feature-name-case": "lower",
+    /* Specify lowercase or uppercase for keywords values. */
+    /* https://stylelint.io/user-guide/rules/value-keyword-case */
+    "value-keyword-case": "lower",
+    /* Disallow extra semicolons. */
+    /* https://stylelint.io/user-guide/rules/no-extra-semicolons */
+    "no-extra-semicolons": true,
+  },
+  ignoreFiles: ["./.cache/**/*", "./node_modules/**/*", "./public/**/*"],
```

Comment:
I would suggest to only apply this to the `node_modules`. The rest is a project specific configuration. Please double-check if stylelint needs to ignore `node_modules` or ignores them by default...